python/src/aiconfig/schema.py

```diff
@@ -59,9 +59,23 @@ class PromptMetadata(BaseModel):
     class Config:
         extra = "allow"
 
+class Attachment(BaseModel):
+    '''
+    Attachment used to pass data in PromptInput for non-text inputs (ex: image, audio)
+    '''
+    # The data representing the attachment
+    data: Any
+    # The MIME type of the result. If not specified, the MIME type will be assumed to be plain text.
+    mime_type: Optional[str] = None
+    # Output metadata
```

Comment:
```suggestion
    # Attachment metadata
```
Assuming this was not intended as metadata specific to an output